tests/system/action/meeting/test_create.py

```diff
@@ -244,3 +252,44 @@ def test_create_permissions(self) -> None:
             }
         )
         self.basic_test({})
+
+    def test_create_with_admin_ids_and_permissions(self) -> None:
+        self.set_models(
+            {
+                "user/1": {
+                    "organization_management_level": None,
+                    "committee_$1_management_level": CommitteeManagementLevel.CAN_MANAGE,
+                }
+            }
+        )
+        meeting = self.basic_test({"admin_ids": [2]})
+        assert meeting.get("user_ids") == [2]
+        admin_group_id = meeting.get("admin_group_id")
+        self.assert_model_exists(
+            "user/2", {f"group_${meeting['id']}_ids": [admin_group_id]}
+        )
+
+    def test_create_with_admin_ids_and_no_permissions(self) -> None:
```

Comment:
Isn't this test redundant with `test_create_no_permissions` above?

Reply:
It has a slightly different payload, but I think we can remove it.